tests/test_custom_item_id.py

```diff
@@ -0,0 +1,34 @@
+from . import test_router
+from pytest import mark
+
+from tests import CUSTOM_TAGS
```

Comment:
You've imported CUSTOM_TAGS but never used it

Reply:
Thanks for the review, removed the import.